app/javascript/components/supplemental_application/sections/preferences/LiveOrWorkInSanFranciscoPanel.js

```diff
@@ -25,12 +25,14 @@ const getLWPrefProofTypeOptions = (individualPrefName) => {
 class LiveOrWorkInSanFranciscoPanel extends React.Component {
   constructor (props) {
     super(props)
+    // NOTE: I do not think we need a state for this, Fed
```

Comment:
@fddayan Let's save refactor of this area for the upcoming story we have that addresses all uses of type of proof in the Confirmed Preferences section.